src/components/common/PageLayout/index.tsx

```diff
@@ -35,6 +36,8 @@ const PageLayout = ({ pathname, children }: { pathname: string; children: ReactE
         })}
       >
         <div className={css.content}>
+          <SocialLoginDeprecation />
```

Comment:
This component is still in the dashboard folder but now used in `PageLayout`.
We should place it there or under` /common`